2019/CG-03-19.md

```diff
@@ -29,6 +29,8 @@ Installation is required, see the calendar invite.
 1. Proposals and discussions
     1. Review of action items from prior meeting.
     1. [WebAssembly ESM integration](https://github.com/WebAssembly/esm-integration/blob/master/proposals/esm-integration/README.md) for Phase 2 ([spec](https://webassembly.github.io/esm-integration/js-api/index.html#esm-integration), [HTML PR](https://github.com/whatwg/html/pull/4372))
+    1. Growing shared WebAssembly.Memory and updating SharedArrayBuffer objects
+    1. [Handling traps as exceptions](https://github.com/WebAssembly/exception-handling/issues/1)
```

Comment:
This does not work for me: https://github.com/WebAssembly/exception-handling/issues/1#issuecomment-471784043

Reply:
Removed.